src/main/scala/cromwell/binding/formatter/SyntaxFormatter.scala

```diff
@@ -133,18 +121,17 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
   }
 
   private def formatWorkflow(workflow: Workflow): String = {
-    val declarations = workflow.declarations.map{formatDeclaration(_, 1)} match {
-      case x: Seq[String] if !x.isEmpty => s"\n${x.mkString("\n")}\n"
-      case _ => ""
-    }
-    s"""${highlighter.keyword("workflow")} ${highlighter.name(workflow.name)} {$declarations
-        |${workflow.calls.map{formatCall(_, 1)}.mkString("\n")}
+    val declarations = workflow.declarations.map{formatDeclaration(_, 1)}
```

Comment:
`declarations.map(...)` or `declarations map { ... }`, the style guide prefers the latter but YMMV